templates/class-live-student.html

```diff
@@ -0,0 +1,97 @@
+{% extends "class-live-stats.html" %}
+
+{% block student_space %}
+
+  <div class="justify-center pt-10 min-h-full">
+      <h2 class="text-center">{{_('Student_Details')}}</h2>
+      <div class="w-full max-w-8xl rounded-lg bg-blue-200 shadow-lg" id="card-student-details">
+        <a class="no-underline float-right p-2"
+           href="/live_stats/class/{{class_info.id}}?{{dashboard_options_args | replace('student='~student.username, 'student=None')}}">
+            ❌
+        </a>
+        <!-- Title columns -->
+        <div class="m-auto ml-14 flex pt-4 pl-4">
+            <div class="flex flex-row">
+              <div class="flex flex-row p-2">
+                  <p class="text-blue-600 font-bold ltr:mr-2 rtl:ml-2">{{_('name')}}</p>
+                  {{student.username}}
+              </div>
+            {% if student_programs %}
+              <div class="flex flex-row p-2">
+                  <p class="text-blue-600 font-bold ltr:mr-2 rtl:ml-2">{{_('level')}}</p>
+                  {{ student_programs[0].level }}
+              </div>
+              <div class="flex flex-row p-2">
+                  <p class="text-blue-600 font-bold ltr:mr-2 rtl:ml-2">{{_('adventure')}}</p>
+                    {% if adventure_names.get(student_programs[0].adventure_name) %}
+                        {{adventure_names.get(student_programs[0].adventure_name)}}
+                    {% else %}
+                        {{student_programs[0].name}}
+                    {% endif %}
+              </div>
+            {% else %}
+                <div class="flex flex-row p-2">
+                    <p class="text-blue-600 font-bold ltr:mr-2 rtl:ml-2">{{_('level')}}</p>
+                </div>
+                <div class="flex flex-row p-2">
+                    <p class="text-blue-600 font-bold ltr:mr-2 rtl:ml-2">{{_('adventure')}}</p>
+                </div>
+            {% endif %}
+            </div>
+        </div>
+
+        <!-- Content -->
+        <div class="grid grid-cols-3 p-5">
+            <!-- Graph with fails and successful runs over time -->
+            <div class="w-full text-center p-2">
+                <h5 class="font-normal ltr:mr-2 rtl:ml-2">{{_('Runs_over_time')}}</h5>
+                <canvas id="runsOverTime" width="120" height="40"></canvas>
+            </div>
+
+
+
+            <!-- Last ran code -->
+            <div class="w-full text-center turn-pre-into-ace mb-4">
+                <h5 class="font-normal pb-1">{{_('Last_program')}}</h5>
+                {% if student_programs %}
+                    <pre level="{{ student_programs[0].level }}">{{ student_programs[0].code }}</pre>
+                {% else %}
+                    <p>No program run yet</p>
```

Comment:
Don't forget to turn this into a babel string

Reply:
Done! (reused a string, does not fit perfectly but I think it is ok)